content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -260,6 +255,19 @@ The above update rules apply to regular pod updates, but other pod fields can be
 - **Binding:** The `binding` subresource allows setting the pod's `spec.nodeName` via a `Binding` request.
   This is typically only used by the {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}}.
 
+### Pod generation
+
+- The `generation` field is unique. It will be automatically set by the
```

Comment:
```suggestion
- The `metadata.generation` field is unique. It will be automatically set by the
```

Previously this was in a section that was talking about metadata, but since we are moving it to its own section, we should make it expicit